crepecake/src/main/java/com/google/cloud/tools/crepecake/blob/Blobs.java

```diff
@@ -20,7 +20,7 @@
 import java.io.InputStream;
 
 /** Static initializers for {@link Blob}. */
-public class Blobs {
+public abstract class Blobs {
```

Comment:
Do we expect people to extend this? Why not a private constructor to prevent instantiation?